cmake/std/atdm/ATDMDevEnvSettings.cmake

```diff
@@ -280,6 +280,7 @@ ATDM_SET_CACHE(Tpetra_INST_SERIAL "${ATDM_INST_SERIAL}" CACHE BOOL)
 ATDM_SET_CACHE(Tpetra_INST_INT_INT OFF CACHE BOOL)
 ATDM_SET_CACHE(Xpetra_ENABLE_Epetra OFF CACHE BOOL)
 ATDM_SET_CACHE(MueLu_ENABLE_Epetra OFF CACHE BOOL)
+ATDM_SET_CACHE(Piro_ENABLE_MueLu OFF CACHE BOOL)
```

Comment:
I doubt very much that SPARC uses Piro.  Not sure about EMPIRE, though.

Reply:
empire does not use piro either.

Reply:
@rppawlo said:

> empire does not use piro either.

Then why are we enabling Piro when we build Trilinos for EMPIRE?  Is it that Panzer enables more tests if it Piro is enabled?

Reply:
Panzer currently has a required dependency on piro for the top level solvers that Drekar and Charon use. Empire does not use this object. Long term we plan to remove piro from panzer, but that requires changing every unit test input file in charon and drekar or replicating the piro ParamList hierarchy in panzer. It's on the list but not a high priority at this point.

Reply:
@rppawlo said:

> Panzer currently has a required dependency on piro for the top level solvers that Drekar and Charon use. Empire does not use this object

Thanks for clarifying.  So we can gut Piro anyway we want to and it will not impact EMPIRE or SPARC.

We don't need to protect Piro (or any other Trilinos package not being used by the ATDM APPs) in ATDM Trilinos builds because those are protected somewhat by the Trilinos PR builds and other customer builds.